### ISSUE TITLE
Show fiat equivalents

### ISSUE THREAD

**Issue by ibz** [PlebeianTech/plebeian-market#1]

**Comment by 151henry151** [comment]
should we maybe pull this data from something like bitcoinaverage.com?

**Comment by ibz** [comment]
@151henry151 I'm making a library for this, see https://github.com/ibz/btc2fiat - the Python version is there, but I was also working on a Javascript version that we can use client-side. Didn't push it yet though.

**Comment by AryanJ-NYC** [comment]
https://github.com/AryanJ-NYC/bitcoin-conversion/

I made a library that might help, guys. :) https://github.com/AryanJ-NYC/bitcoin-conversion/

**Comment by AryanJ-NYC** [comment]
I got your app running locally (y'all made it SUPER EASY).

<img width="649" alt="Screen Shot 2022-06-24 at 7 30 23 PM" src="https://user-images.githubusercontent.com/8263430/175746092-96929a1e-bee3-4382-98f9-2ebda82d730f.png">


Would y'all like any help here by chance? I'd love to take a stab at it. I've never used Svelte but am super down to give it a shot.

If you're cool with me taking a stab, mind if I use that library I linked above?

**Comment by ibz** [comment]
Hey! Your library looks good, but I really wanted to use my library for multiple reasons (I gave it quite a bit of thought when I built it). One reason is that it can fetch data from multiple exchanges - I noticed that over time one can't rely on just one API, as it might go away or break. Also, conversion to multiple currencies is something I would not want in such a library, since that is (sometimes) an extra step - some exchanges provide only USD/EUR and there needs to be an extra conversion between fiat currencies - I see that's not the case in your library, since it all comes from Coindesk, but that also means that if I later want to switch to using another exchange, an extra layer of complexity would need to be added to the library.

Really excited to hear you liked our project! I am a huge fan of your stuff as well. In fact, your blog post about how scarce.city is built helped me a lot - that's how I discovered Tailwind CSS!
